### ISSUE TITLE
Website breaks when inserting inline math into specific text

### ISSUE THREAD

**Issue by pxnt** [imzbf/md-editor-v3#72]
During testing I used the following text and encountered some sort of overload, causing the site to basically stop responding entirely.

To replicate, copy this markdown text:

```md
## What is Lorem Ipsum?

**Lorem Ipsum** is simply dummy text of the printing and typesetting industry.
```

and then insert an inline math block (`$$`) by hand or using the button for it between the header and the text, like this:

```md
## What is Lorem Ipsum?
$$
**Lorem Ipsum** is simply dummy text of the printing and typesetting industry.
```

This happens on the [demo page](https://imzbf.github.io/md-editor-v3/index) as well as on my test build (running version 1.11.4). It seems to happen when any header follows a few words, and the inline math is inserted in between.

**Comment by imzbf** [comment]
Thanks, I am trying to fix it~

**Comment by imzbf** [comment]
It was fixed in `v1.11.5`, have a try~
